pygmt/src/blockm.py

```diff
@@ -147,6 +147,16 @@ def blockmean(data=None, x=None, y=None, z=None, outfile=None, **kwargs):
           is not set.
         - None if ``outfile`` is set (filtered output will be stored in file
           set by ``outfile``).
+
+    Example
+    -------
+    >>> import pygmt  # doctest: +SKIP
+    >>> # Load a table of ship observations of bathymetry off Baja California
+    >>> data = pygmt.datasets.load_sample_bathymetry()
```

Comment:
```suggestion
    >>> data = pygmt.datasets.load_sample_data(name='bathymetry')
```
To avoid the deprecation warning.